reactions/src/SPT.cpp

```diff
@@ -781,22 +782,20 @@ static double ploopn2_mgdd_nu( const PowerSpectrum& P_L, double vars[], double k
         kargs[2] = sqrt(kv[1]*kv[1]+2.*kv[1]*kv[2]*xv[0]+kv[2]*kv[2]);
         kargs[1] = sqrt(kv[2]*kv[2]+2.*kv[2]*kv[0]*xv[2]+kv[0]*kv[0]);
         kargs[3] = sqrt(kv[0]*kv[0]+2.*kv[0]*kv[1]*xv[1]+kv[1]*kv[1]);
-        iow.initn2(vars[0],kv,xv,kargs,omegacb,vars[2],vars[3],vars[4],vars[6]);
+        iow.initn2(vars[0],kv,xv,kargs,vars[1],vars[2],vars[3],vars[4],vars[6]);
```

Comment:
Can you remind me why we now use Omega_m instead of Omega_cb here? I can't see any change to initn2 that matches this, although I see vars[6] is now Omega_cb.

Reply:
So we calculate omega_cb in funcn1 and funcn1_pseudo in SpecialFunctions.cpp using omega0 and omeganu.

Reply:
Ah right, for some reason I thought vars[6] was Omega_cb but as you say it is Omega_nu - my bad!